client/my-sites/current-site/index.jsx

```diff
@@ -113,7 +113,6 @@ const CurrentSite = React.createClass( {
 	},
 
 	previewSite: function( event ) {
-		analytics.ga.recordEvent( 'Sidebar', 'Clicked View Site' );
```

Comment:
@folletto Let's handle anything like new Tracks in a separate PR. If you open an issue, feel free to assign me.